schema.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE animals (
+  id int PRIMARY KEY,
```

Comment:
- [OPTIONAL] Good job but you can add not null here and also `SERIEL` keyword so that the id will auto increment like this : `id SERIAL   NOT NULL PRIMARY KEY,`

Reply:
Thanks for the feedback@Zuhaib042